build/source/driver/multi_driver.f90

```diff
@@ -515,16 +483,16 @@ program multi_driver
 
 ! set default model parameters
 do iGRU=1,nGRU
- do localHRU=1,gru_struc(iGRU)%hruCount
+ do iHRU=1,gru_struc(iGRU)%hruCount
```

Comment:
After the merge, half the lines in `multi_driver.f90` used iHUR and half used localHRU. I had to choose one, and I went with iHRU because it is intuitive, it matches the other loop syntax in the rest of the code, and iHRU specifically is used elsewhere in the code.